soundata/datasets/tau2021sse_nigens.py

```diff
@@ -3,7 +3,7 @@
 .. admonition:: Dataset Info
     :class: dropdown
 
-    *TAU NIGENS Spatial Sound Events: scene recordings with (moving) sound events of distinct categories*
+    **TAU NIGENS Spatial Sound Events: scene recordings with (moving) sound events of distinct categories**
 
     *Created By:*
         | Archontis Politis, Sharath Adavanne, Tuomas Virtanen.
```

Comment:
Remove `|` as in [dcase23_task4b.py](https://github.com/soundata/soundata/pull/165/files#diff-ea6737eb033c8c11fd6da408f6e73877aad3dbad484c1079a466d4dc6b398707)